.github/workflows/astarte-housekeeping-api-workflow.yaml

```
@@ -0,0 +1,35 @@
+name: Build and Test Astarte Housekeeping API
+
+on:
+  # Run when pushing to stable branches
+  push:
+    paths:
+    - 'apps/astarte_housekeeping_api'
+    - '.github/workflows/astarte-apps-build-workflow.yaml'
+    - '.github/workflows/astarte-housekeeping-api-workflow.yaml'
```

We should run the tests for the API modules even when only the backend module was modified

```suggestion
    paths:
    - 'apps/astarte_housekeeping'
    - 'apps/astarte_housekeeping_api'
    - '.github/workflows/astarte-apps-build-workflow.yaml'
    - '.github/workflows/astarte-housekeeping-api-workflow.yaml'
```

same for all other api modules